tutorials-and-examples/workflow-orchestration/dws-examples/dws-queues.yaml

```diff
@@ -28,20 +28,20 @@ kind: ClusterQueue
 metadata:
   name: "dws-cluster-queue"
 spec:
-  namespaceSelector: {} 
+  namespaceSelector: {}
   resourceGroups:
   - coveredResources: ["cpu", "memory", "nvidia.com/gpu", "ephemeral-storage"]
     flavors:
     - name: "default-flavor"
       resources:
       - name: "cpu"
-        nominalQuota: 10000  # Infinite quota.
+        nominalQuota: 1000000000    # "Infinite" quota
       - name: "memory"
-        nominalQuota: 10000Gi # Infinite quota.
+        nominalQuota: 1000000000Gi  # "Infinite" quota
       - name: "nvidia.com/gpu"
-        nominalQuota: 10000  # Infinite quota.
+        nominalQuota: 1000000000    # "Infinite" quota
       - name: "ephemeral-storage"
-        nominalQuota: 10000Ti # Infinite quota.
+        nominalQuota: 1000000000Ti  # "Infinite" quota
```

Comment:
Have you manually tested that this works?

Reply:
Yes